content/tr/abstraction.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Soyutlama (Abstraction)
```

Comment:
One thing that I'm worried is that if the title is somewhat long, with the parentheses it'll be hard to read and probably won't look good. My suggestion is to keep the title in Turkish and include the English equivalent where we mention it first time in text, probably in the first paragraph.

Would love to hear your thoughts.

Reply:
I like your suggestion, makes sense.